test/FBeetsRevenueSharer.test.ts

```diff
@@ -123,7 +125,84 @@ describe("BeetsBar", function () {
     expect(await beets.balanceOf(bob.address)).to.be.closeTo(userInfo.rewardDebt.div(2), 200000)
   })
 
-  it("distributes 50% of the emissions to all fBeets holders in the form of fBeets revenue share", async () => {
+  it("change fBeets locker share above 100%", async () => {
```

Comment:
when you think of use case, the title kinda suggests that there is a use case to change locker share above 100%. but actuall the use case is that it `reverts when changing the locker share above 100%`